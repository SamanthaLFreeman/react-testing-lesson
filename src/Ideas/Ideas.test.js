import React from 'react'
import { shallow } from 'enzyme'
import Ideas from './Ideas'


describe('Ideas', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<Ideas
      ideas={[{id:1, title: "Blah", description: "Blah blah blah"}]}
      removeIdea={jest.fn()}
    />)

    expect(wrapper).toMatchSnapshot()
  })
})