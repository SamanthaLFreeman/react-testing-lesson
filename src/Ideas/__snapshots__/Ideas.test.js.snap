// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ideas should match the snapshot with all data passed in correctly 1`] = `
<Fragment>
  <h2>
    Ideas Component
  </h2>
  <section
    className="ideas"
  >
    <Card
      description="Blah blah blah"
      id={1}
      key="1"
      removeIdea={[MockFunction]}
      title="Blah"
    />
  </section>
</Fragment>
`;
